### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by tefkah** [org-roam/org-roam-ui#99] [Product]
If you have any thoughts on how org-roam-ui functions please let us know here!

While the other discussions are more feature focused, we are also interested in what you think about the smaller minutia: is the graph snappy enough, do you like the look of the ui, etc, feel free to be as nitpicky as possible.

**Comment by matthew-piziak** [comment]
@ThomasFKJorna Thank you for being so encouraging! I've written up a few thoughts on graph layout [here](https://github.com/org-roam/org-roam-ui/issues/100). It's highly speculative: more of a request for perspective than a real feature request. Is there a better place to post half-baked musings than the issue tracker? Thank you!

**Reply by tefkah** [reply]
Thanks for your thoughts! I'll respond in more detail on the issue (or discussion), but I really like the idea!
I think the discussions are a perfect place for musings! Please feel free to open a new topic for such things :)

**Comment by buhtz** [comment]
I have a question.

Is the running org-roam-ui server accessible from another computer in the same network?

It doesn't work here but it could be related to my own network settings independend from org-roam-ui configuration. Or is there a config option for org-roam-ui I need to activate first?

**Reply by tefkah** [reply]
I think this is probably the same issue as #204, see my response there!

**Comment by buhtz** [comment]
Yes it is related to #204.

Close here. #Close

**Comment by ashton314** [comment]
Holy crap!!! I turned this on and I nearly fell out of my chair! I understand it's alpha and a bit buggy, but it Just Worked :tm: for me and I was blown away by the following, the visualization, and the "view in Emacs". Dang. Very impressive, very delightful. Thank you for sparking some serious joy in my Emacs life tonight. :clap: I'm gonna show this to every person who thinks Emacs is anything less than the GOAT. Thank you!!

(Sorry if this is OT. I had to express my gratitude somewhere.)
